appstream.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>ml.prevete.Daty</id>
+    <launchable type="desktop-id">ml.prevete.Daty.desktop</launchable>
+    <name>Daty</name>
+    <summary>Free Wikidata editor.</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <description>
+        <p>
+            Daty is a Wikidata editor
+        </p>
+        With it you can search for concepts and read their properties.
+        <p>
+            Progressive layout
+        </p>
+        This application is ready for gnome mobile usage.
+        <p>
+            This is a preview release!
+        </p>
+        See the list of <a href="https://prevete.ml/articles/daty.html">planned features</a> for v1.0 stable!
+    </description>
+    <url type="homepage">https://gitlab.gnome.org/World/Daty/</url>
+    <url type="help">https://gitlab.gnome.org/World/Daty/#documentation</url>
+    <url type="bugtracker">https://gitlab.gnome.org/World/Daty/issues</url>
+    <url type="donation">http://patreon.com/tallero</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://gitlab.gnome.org/World/Daty/raw/master/screenshots/editor.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://gitlab.gnome.org/World/Daty/raw/master/screenshots/mobile-page-view.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://gitlab.gnome.org/World/Daty/raw/master/screenshots/mobile-view-sidebar.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://gitlab.gnome.org/World/Daty/raw/master/screenshots/free-text-search.png</image>
+        </screenshot>
+    </screenshots>
+    <releases>
+        <release version="1.0a" date="2019-01-25">
+            <description>
+                <p>Preview: Reader functions only.</p>
+            </description>
+        </release>
+    </releases>
+</component>
```

Comment:
Also needs OARS rating: https://hughsie.github.io/oars/

Ideally it should be shipped with your code so other distributions can also use it but it's okay for now.